### DISCUSSION TITLE
Accessing bundle identifier from JavaScript.

### DISCUSSION THREAD

**Discussion by aleph1** [tauri-apps/tauri#6752] [Q&A]
Does Tauri expose the bundle identifier in a way that it is accessible from JavaScript?

**Comment by DerrickBarra** [comment]
Haven't spotted anything for this in the API myself. It would be incredibly useful to have access to this.

**Comment by hanaTsuk1** [comment]
There is an indirect way to get it
https://tauri.app/zh-cn/v1/api/js/path#appdatadir
